source/site-administrators/studio/rte2-setup.rst

```diff
@@ -148,6 +148,107 @@ You can also add custom elements to the rule set and can be done by simply addin
       :width: 85%
       :align: center
 
+   |
+
+.. _adding-external-plugins:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Adding External Plugins
+^^^^^^^^^^^^^^^^^^^^^^^
+
+TinyMCE provides an option to specify URLS to plugins outside the tinymce plugins directory.  These external plugins allows the user to extend TinyMCE.  For example, you can create custom dialogs, buttons, menu items, etc.
+
+To add an external plugin, use the tag ``<external_plugins />`` in the RTE configuration.  Use the Crafter Studio API that gets a file for a given plugin, the getPluginFile API found here https://app.swaggerhub.com/apis/craftercms/studio/3.1.14.0#/plugin/getPluginFile to get the Tiny MCE external plugin file to pass to the RTE.
+
+Example External Plugin
+^^^^^^^^^^^^^^^^^^^^^^^
+Let's take a look at an example of a simple external plugin that creates a custom button which inserts text in the RTE.
+We'll then load our external plugin then add the custom button to the RTE's toolbar.  For our example, we'll be using a site created using the empty blueprint named ``hello``.
+
+1. Open the RTE (TinyMCE 5) configuration file in Studio by opening the **Sidebar**, then click on |siteConfig| -> *Configuration* -> *RTE (TinyMCE 5) Configuration*
+
+2. First, we'll load the external plugin using the ``<external_plugins />`` tag and use the getPluginFile API to get the external plugin file for the RTE |br|
+
+   .. code-block:: xml
+
+      <external_plugins>
+        <my_button><![CDATA[/studio/api/2/plugin/file?siteId=hello&type=tinymce&name=my_button&filename=plugin.min.js]]></my_button>
+      </external_plugins>
+
+   |
+
+3. Next, we'll add the custom button we're creating to the toolbar of the RTE.  Scroll down to ``<toolbarItems2 />`` tag and add the custom button we are creating ``my_button``
```

Comment:
The toolbarItems are actually above `external_plugins` in the default config files. Since this is presented as the second step, I'd think the reader is at `external_plugins`; easier if we just omit up/down so it doesn't matter where they are.
```suggestion
3. Next, we'll add the custom button we're creating to the toolbar of the RTE.  Scroll to ``<toolbarItems2 />`` tag and add the custom button we are creating ``my_button``
```